client/src/contentful-common-strings/pull.js

```diff
@@ -0,0 +1,71 @@
+//@ts-check
+
+/**
+ * @typedef {import("./index").ContentfulCommonStrings} CommonStrings
+ */
+
+let https = require("https");
+
+const ORIGIN = "https://cdn.contentful.com";
+const COMMON_STRING_TAG = "common";
+const SPACE_ID = "markmr2gi204";
+const ACCESS_TOKEN = "Fli_OMdKgUFw6tEX3uv6HqvptuG6A6jn9bZVPlHZj8E";
```

Comment:
Ah, this was what the email was complaining about?

Reply:
yupppppppp 😠